Source/Forms/Gen4/Wild4.cpp

```diff
@@ -435,6 +426,18 @@ void Wild4::generatorEncounterIndexChanged(int index)
             break;
         }
 
+        if ((currentProfile->getVersion() & Game::HGSS) != Game::None && (encounter == Encounter::OldRod || encounter == Encounter::GoodRod || encounter == Encounter::SuperRod))
+        {
+            if (!ui->toolButtonGeneratorLead->findAction(tr("Suction Cups")))
+            {
+                ui->toolButtonGeneratorLead->addAction(tr("Suction Cups"), toInt(Lead::SuctionCups));
+            }
+        }
+        else if (ui->toolButtonGeneratorLead->findAction(tr("Suction Cups")))
```

Comment:
I don't think the check here is necessary. Remove already internally checks if the item exists

Reply:
reverted